go/kbfs/simplefs/simplefs.go

```diff
@@ -861,6 +861,10 @@ func (k *SimpleFS) SimpleFSReadList(_ context.Context, opid keybase1.OpID) (keyb
 	return lr, nil
 }
 
+// SimpleFSListFavorites lists the favorite, new,
+// and ignored folders of the logged in user,
+// getting its data from the KBFS Favorites cache. If the cache is stale,
```

Comment:
Inconsistent line wrapping.

Reply:
Whoops, I merged after strib's approve and then saw your comment just after clicking merge. Will fix later. My editor really likes to wrap comments on punctuation :/